README.md

```diff
@@ -190,13 +196,16 @@ History
 
 See the git log for details at http://github.com/libffi/libffi.
 
-    3.3 TBD
+    3.4 TBD
```

Comment:
Please merge this with 3.3.  We're not making a new release for this (and will try to get out 3.3 soon)

Reply:
have incorporated your suggestion.